starter-repos/app-template/config/envs/dev/kustomization.yaml

```diff
@@ -40,23 +36,3 @@ vars:
 
 configurations:
 - varReference.yaml
```

Comment:
These are repeating configs in all environment. Does it make sense to move `configGenerator`, `vars`, and `configurations` to the base? @Liujingfang1

Reply:
Most likely yes, I'll change it.

Reply:
Here is the change https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/pull/95